src/backend/distributed/planner/multi_logical_planner.c

```diff
@@ -1828,6 +1815,7 @@ MultiProjectNode(List *targetEntryList)
 MultiExtendedOp *
 MultiExtendedOpNode(Query *queryTree)
 {
+	StringInfo errorInfo = NULL;
```

Comment:
we are not user `errorInfo`, can we make `SafeToPushdownWindowFunction` to accept a `NULL` pointer as the second argument which skips the writing of the `errorInfo` if we are omitting it anyway?